Corrfunc/theory/vpf.py

```diff
@@ -200,8 +200,12 @@ def vpf(rmax, nbins, nspheres, numpN, seed,
         msg = "Number of counts-in-cells wanted must be at least 1"
         raise ValueError(msg)
 
-    if periodic and boxsize is None:
-        raise ValueError("Must specify a boxsize if periodic=True")
```

Comment:
Writing out this way seems easier to understand - any reason for moving this check into the kwargs section?

Reply:
It was just a style preference; having two adjacent conditionals that share a boolean to me says they could/should be merged! But I don't disagree that this way is more explicit.